addons/medical_feedback/functions/fnc_effectBloodVolumeIcon.sqf

```diff
@@ -0,0 +1,44 @@
+#include "script_component.hpp"
+/*
+ * Author: 10Dozen
+ * Handles the blood volume icon.
+ *
+ * Arguments:
+ * 0: Enable <BOOL>
+ * 1: Intensity 0...6 <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [true, 4] call ace_medical_feedback_fnc_effectBloodVolumeIcon
+ *
+ * Public: No
+ */
+
+params ["_enable", "_intensity"];
+
+private _indicatorCtrl = uiNamespace getVariable [QGVAR(bloodVolumeIndicator), controlNull];
+if (isNull _indicatorCtrl) exitWith {};
```

Comment:
This check is superfluous.

Reply:
Why? All below is worthless if control is null, so why to keep doing it?

Reply:
The control is 99-100% not null, so why check it, if - when the check is left out - it makes no practical difference?

All this check does is make the code slower on average.

Reply:
Removed